caravel/config.py

```diff
@@ -210,6 +210,12 @@ class CeleryConfig(object):
 SQL_CELERY_DB_FILE_PATH = os.path.join(DATA_DIR, 'celerydb.sqlite')
 SQL_CELERY_RESULTS_DB_FILE_PATH = os.path.join(DATA_DIR, 'celery_results.sqlite')
 
+# static http headers to be served by your Caravel server.
+# The following example prevents iFrame from other domains
+# and "clickjacking" as a result
+# HTTP_HEADERS = {'X-Frame-Options': 'SAMEORIGIN'}
```

Comment:
It's there as an example, I don't want to set headers on behalf of others. People override this configuration by setting up a `caravel_config` module in their PYTHONPATH (for us it's doen in chef), where I have another PR.